src/components/transport_manager/include/transport_manager/iap2_emulation/iap2_transport_adapter.h

```diff
@@ -60,9 +60,13 @@ class IAP2BluetoothEmulationTransportAdapter : public TcpTransportAdapter {
    */
   IAP2BluetoothEmulationTransportAdapter(
       const uint16_t port,
-      resumption::LastState& last_state,
+      resumption::LastStateWrapperPtr last_state_wrapper,
       const TransportManagerSettings& settings);
 
+  IAP2BluetoothEmulationTransportAdapter(
```

Comment:
```suggestion
  DEPRECATED IAP2BluetoothEmulationTransportAdapter(
```
?

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508